### PR TITLE
Generate DICE data in the orchestrator

### PR DESCRIPTION
**Pull request by conradgrobler**
Instead of adding a new layer of evidence the Orchestrator generates the Application Keys (nd encryption key and a signing key) and adds their certificates to the evidence.

Ref https://github.com/project-oak/oak/issues/4074

### CODE REVIEW

File: oak_containers_orchestrator/src/main.rs [hunk 1]
```diff
@@ -58,8 +59,21 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
...
     let encryption_key_provider = Arc::new(EncryptionKeyProvider::new());
+    // Ignore the signing key for now.
+    let (_signing_key, verifying_key) = generate_ecdsa_key_pair();
+
+    let dice_evidence = dice_builder.add_application_keys(
+        additional_claims,
+        &encryption_key_provider.get_serialized_public_key(),
+        &verifying_key,
+    )?;
+    let attestation_report_generator = Arc::new(EmptyAttestationReportGenerator);
```
Comment (ipetr0v) [inline oak_containers_orchestrator/src/main.rs:76]: Could you please add a TODO(https://github.com/project-oak/oak/issues/4074) that this is to be removed once DICE attestation is implemented?

File: oak_containers_orchestrator/src/ipc_server.rs [hunk 2]
```diff
@@ -54,19 +51,6 @@ impl Orchestrator for ServiceImplementation {
         Ok(tonic::Response::new(()))
     }
 
-    async fn get_attestation_evidence(
```
Comment (ipetr0v) [inline oak_containers_orchestrator/src/ipc_server.rs:57]: Does it mean that the enclave app won't be able to request it's own evidence?

**Review by ipetr0v — APPROVED** [review]

File: oak_containers_orchestrator/src/main.rs (hunk 1 above)
Comment (ipetr0v) [reply oak_containers_orchestrator/src/main.rs:76]: Maybe in the next PR to not rerun the CI

Comment (conradgrobler) [reply oak_containers_orchestrator/src/main.rs:76]: Done.

File: oak_containers_orchestrator/src/ipc_server.rs (hunk 2 above)
Comment (conradgrobler) [reply oak_containers_orchestrator/src/ipc_server.rs:57]: Yes, I don't think we have any plans that require this on Oak Containers.

**Review by conradgrobler — COMMENTED** [review]
Thanks.

**Pull request merged**